pipelines/docker-compose-for-test.yaml

```diff
@@ -0,0 +1,113 @@
+version: '3.9'
+
+networks:
+  badgerdoc-scheduler:
+    driver: bridge
+
+services:
+  minio:
+    image: 'bitnami/minio:latest'
+    ports:
+      - "9000:9000"
+    env_file:
+      - .env
+
+  db:
+    image: postgres:13
+    volumes:
+      - ./pg-init-scripts:/docker-entrypoint-initdb.d
+      - pgdata:/var/lib/postgresql/data
+    networks:
+      - badgerdoc-scheduler
+    ports:
+      - "5432:5432"
+    env_file:
+      - .env
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_MULTIPLE_DATABASES=pipelines,scheduler,file_management,annotation,jobs
+
+  pgadmin:
+    image: dpage/pgadmin4
+    restart: on-failure
+    environment:
+      PGADMIN_DEFAULT_EMAIL: admin@admin.com
+      PGADMIN_DEFAULT_PASSWORD: admin
+      PGADMIN_CONFIG_CONSOLE_LOG_LEVEL: 250
+    ports:
+      - "5050:80"
+
+  zookeeper:
+    image: wurstmeister/zookeeper
+    container_name: zookeeper
+    ports:
+      - "2181:2181"
+    logging:
+      driver: none
+    networks:
+      - badgerdoc-scheduler
+
+  kafka:
+    image: wurstmeister/kafka
+    container_name: kafka
+    ports:
+      - "9092:9092"
+    logging:
+      driver: none
+    environment:
+      KAFKA_ADVERTISED_HOST_NAME: kafka
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_AUTO_CREATE_TOPICS_ENABLE: 'false'
+    networks:
+      - badgerdoc-scheduler
+    depends_on:
+      - zookeeper
+
+  pipelines:
+    build:
+      context: .
+      target: development
+    env_file:
+      - .env
+    environment:
+      - DB_URI=postgresql+psycopg2://postgres:postgres@db:5432/pipelines
+      - S3_CREDENTIALS_PROVIDER=minio
+      - S3_ENDPOINT=minio:9000
+      - S3_ACCESS_KEY=minioadmin
+      - S3_SECRET_KEY=minioadmin
+      - PREPROCESSING_URL=//preprocess:8000/
+      - POSTPROCESSING_URI=//postprocess:8004/
```

Comment:
Are these slashes at the beginning and the end intentional?

Reply:
They are.
I've copied it from existing (but not entirely properly working - that's why I don't use it) docker-compose.yaml.

A bit more about this syntax:
https://stackoverflow.com/questions/9646407/two-forward-slashes-in-a-url-src-href-attribute#:~:text=The%20%22two%20forward%20slashes%22%20are,to%20load%20the%20current%20page%22.&text=type%20of%20logic%20all%20over,through%20both%20http%20and%20https%20).